blogs/13_bfcl_v3_multi_turn.html

```diff
@@ -748,73 +751,59 @@ <h4>5. API Code Validation (🧑‍💻+ 💻 )</h4>
 
         </div>
         <div>
-          <h3 id="inference">Multi-turn Model Inference and Execution</h3>
-          <p>In <i>BFCL V3 • Multi-Turn & Multi-Step</i>, we evaluate multi-turn function-calling models through two types of models: Function-Calling
-            (FC) models and prompting models. The distinction lies primarily in how the models generate outputs and
-            how we handle those outputs during the inference process. This section explains the implementation behind
-            model inference and how multi-turn interactions are managed, including the differences between various
-            model types and the steps taken to ensure correct function execution and state-based evaluation.</p>
-
-          <h4>1. Differences in Inference Patterns Between FC and Prompting Models</h4>
-          <p>For FC models, the output is generated in JSON format, where each function call is structured with clear
-            parameters and ready for direct execution by the API backend.</p>
-          <p>In contrast, prompting models generate outputs as function call strings. For instance, the model may
-            output a function call in the following format:</p>
-          <pre><code>"[write_to_file(filename='log.txt', content='hello'), close_file(filename='log.txt')]".</code></pre>
-          <p>These outputs require parsing before they can be executed. The string format is first parsed into
-            individual function calls, and each function call is then executed in the backend as if it were a normal
-            function-calling API.</p>
-          <p>The key distinction is that FC models produce structured, executable JSON directly, while prompting
-            models require post-processing to interpret and extract the function calls from the string format.</p>
-
-          <h4>2. Handling Different Multi-Turn Function Call Patterns</h4>
-          <p>Multi-turn function calling can present a variety of challenges in inference, particularly when it comes
-            to managing the flow of data and function results across multiple steps. In <i>BFCL V3 • Multi-Turn & Multi-Step</i>, our model handlers
-            are designed to handle different function call patterns—simple, parallel, and nested—across multiple
-            rounds of interaction. The distinction between these call patterns is explained in the previous section.
+          <h3 id="inference">Model Inference & Execution</h3>
```

Comment:
Let's keep a backlog for this part, create a diagram as well for this section as next step.